src/nodes/maths/vector3_math.gd

```diff
@@ -45,3 +46,26 @@ func _generate_outputs() -> void:
 			output[0] = a.floor()
 		"Ceil":
 			output[0] = a.ceil()
+			
+func _on_dropdown(i, j):
+	var operation: String = get_input_single(0, "Add")
+	
+	match operation:
+		"Add": 
+			set_output(0, "", ConceptGraphDataType.VECTOR3)
+		"Substract": 
+			set_output(0, "", ConceptGraphDataType.VECTOR3)
+		"Cross product": 
+			set_output(0, "", ConceptGraphDataType.VECTOR3)
+		"Dot product": 
+			set_output(0, "", ConceptGraphDataType.SCALAR)
+		"Bounce": 
+			set_output(0, "", ConceptGraphDataType.VECTOR3)
+		"Normalize": 
+			set_output(0, "", ConceptGraphDataType.VECTOR3)
```

Comment:
Here for example, the second vector input is not used so you can call remove_input(2) to only leave one

But that also means we need to call  set_input(2, "B", ConceptGraphDataType.VECTOR3, opts) everywhere two inputs are needed